vmm/src/cpu.rs

```diff
@@ -432,11 +443,29 @@ impl Vcpu {
         {
             kvi.features[0] |= 1 << kvm_bindings::KVM_ARM_VCPU_PMU_V3;
         }
+
+        if config.features.sve
+            && sve_supported
+            && vm
+                .as_any()
+                .downcast_ref::<hypervisor::kvm::KvmVm>()
```

Comment:
@jinankjain I think there is quite a bit of KVM-ism in this aarch64 code path. I don't think we want to block the progress on the KVM front though. We can refactor the code when our support is added.

Reply:
Yes this is already cleaned up in my tree for ARM support on MSHV. I just need to post those patches.

Reply:
If SVE is enabled where available by default then this can be moved to the hypervisor crate.